content/actions/index.md

```diff
@@ -48,4 +48,11 @@ body = '''
 Manage teams and team membership for the Scientific Python GitHub organization.
 '''
 
+[[item]]
+type = 'card'
+header = '[circleci-artifacts-redirector-action](https://github.com/scientific-python/circleci-artifacts-redirector-action)'
+body = '''
+Add a GitHub Status link directly to a CircleCI artifact.
```

Comment:
For consistency with the project readme
![image](https://github.com/scientific-python/tools.scientific-python.org/assets/60778417/7a1442db-4333-4f33-993b-17b0882d6d7d)

```suggestion
Add a GitHub status link directly to a CircleCI artifact.
```